projects/fsastorefrontlib/src/cms-components/checkout/components/cart-coupon/cart-coupon.component.html

```diff
@@ -0,0 +1,39 @@
+<ng-container *ngIf="cart$ | async as cart">
+  <ng-container
+    *ngIf="
+      cart.insuranceQuote?.state?.code !== 'BIND' &&
+      (userRegistered$ | async) === currentUser
```

Comment:
Move this check to ts file?

Reply:
I would leave it like this, since this way observable is handled by Angular template.

Reply:
Right, i missed that async pipe